src/pages/getting-started/manage_content/index.md

```diff
@@ -17,7 +17,7 @@ A partner application can affiliate content in Lightroom with content in its own
 
 ## Uploading to Lightroom and Managing Content
 
-_Note: When a [new asset is created and uploaded](./upload.md) to a Lightroom catalog, partner applications should always set the `importedOnDevice` field to their API key. This will ensure that the asset is properly tagged in Lightroom as having originated from the partner application. They should also retain the unique identifier of the asset (`asset_id`) and catalog (`catalog_id`) for use in the workflows detailed below._
+_Note: When a [new asset is created and uploaded](getting-started/upload_content/) to a Lightroom catalog, partner applications should always set the `importedOnDevice` field to their API key. This will ensure that the asset is properly tagged in Lightroom as having originated from the partner application. They should also retain the unique identifier of the asset (`asset_id`) and catalog (`catalog_id`) for use in the workflows detailed below._
```

Comment:
This should be changed to relative as well

Reply:
Done.